File: 1. Matrix Input & Output.cpp
// Topic: Matrix Input & Output
#include <bits/stdc++.h>
using namespace std;

int main()
{
    ios::sync_with_stdio(false);
    cin.tie(nullptr);
    int n = 3, m = 3;
    vector<vector<int>> A = {{1, 2, 3}, {4, 5, 6}, {7, 8, 9}};
    cout << "Matrix:\n";
    for (auto& r : A)
    {
        for (auto v : r) cout << setw(4) << v;
        cout << "\n";
    }
    return 0;
}

// TC: O(n*m)
// SC: O(1) extra

File: 1. Matrix Input & Output.py
"""Topic: Matrix Input & Output
"""
def print_matrix(A):
    for row in A:
        print(' '.join(f"{v:4}" for v in row))

if __name__ == '__main__':
    A = [[1,2,3],[4,5,6],[7,8,9]]
    print("Matrix:")
    print_matrix(A)

# TC: O(n*m)
# SC: O(1) extra
